api/src/main/resources/workbench-api.yaml

```diff
@@ -1128,6 +1128,33 @@ paths:
           description: No module exists with name submitted
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/admin/users/updateAccount":
+    post:
+      tags:
+        - profile
+      consumes:
+        - application/json
+      description: 'Updates a subset of a user''s account metadata, as defined by the AccountPropertyUpdate object.  Requires ACCESS_CONTROL_ADMIN authority.'
+      parameters:
+        - in: body
+          name: request
+          description: The admin edit request to perform on a user
+          schema:
+            "$ref": "#/definitions/AccountPropertyUpdate"
+      operationId: updateAccountProperties
+      responses:
+        200:
```

Comment:
aside: we should start using a yaml partial to bring these common codes in if we want to declare them everywhere. I'm not sure there's much percentage in doing it for all the endpoints.